pylutron/__init__.py

```diff
@@ -737,6 +742,18 @@ def level(self, new_level):
 #    self._lutron.send(Lutron.OP_EXECUTE, Output._CMD_TYPE,
 #        Output._ACTION_ZONE_LEVEL, new_level, fade_time, delay)
 
+  def start_raising(self):
```

Comment:
I'm using the naming in the docs.
<img width="892" alt="Screenshot 2020-05-12 at 17 26 25" src="https://user-images.githubusercontent.com/29941924/81713079-b95a2a80-9475-11ea-95f1-223fc52f9d7e.png">

Reply:
Are these used on anything other than Shades?  The Shade class already implements this.

Reply:
The are Lutron Motors to control not Lutron shades and covers. In fact I have 15 motors ;)

Reply:
Looking at documentation, jogging is ONLY in Homeworks, so it would make more sense to me to have:

class Motor(Shade):

And add jogging there.

Reply:
done